src/index.html

```diff
@@ -0,0 +1,240 @@
+      <ol>
+        <li><a href="#game-rules">Game Rules</a></li>
+        <li><a href="#regulations">Regulations</a></li>
+        <li><a href="#specifications">Specifications</a></li>
+        <li><a href="#comp-structure">Competition Structure</a></li>
+      </ol>
+    </section>
+    <section>
+      <h2><a class="anchor" id="game-rules"></a>Game Rules</h2>
+      <ol>
+        <li>
+          The objective of the game, called <strong>Tin Can Rally</strong>, is to race around a track, picking up tin cans along the way.
+        <li>
+          Before a match begins, participating teams must:
+          <ul>
+            <li>Present their robot in the staging area, adjacent to the arena, before the scheduled close of staging time. The staging area will be clearly marked on the day.
+            <li>Attach a <a href="#flag-specification">robot flag</a>. Robot flags will be provided by SourceBots officials in the staging area.
+            <li>Place their robot in the starting area that they are assigned. The robot must be placed such that it is entirely within this starting area, with no parts overhanging its boundary. Its orientation does not matter.
+            <li>Vacate the arena 40 seconds before the scheduled start time. During the 40 second period prior to the start of the match there must be no interaction with the robot.
+            <li>Follow the directions of the match officials.
+          </ul>
+          Teams that fail to comply with these rules – such as by arriving late – may forfeit the match, at the discretion of the judge.
+        <li>
+          The game is played between <strong>two</strong> robots.
+        <li>
+          Each match lasts for <strong>150</strong> seconds.
+        <li>
+          Robots will be started by, or at the direction of, match officials.
+        <li>
+          A point is awarded each time a robot crosses a scoring line in the anti-clockwise direction.
+        <li>
+          Robots can pick up tin cans which are in the track. For every can a robot is carrying when it crosses a scoring line, the number of points awarded is doubled.<br>For example, a robot carrying two cans is awarded four points.
+        <li>
+          At the end of each lap - defined as every sixth crossing of a scoring line in a forward direction - a robot is awarded an additional four points above the points it would normally get for the crossing.
+        <li>
+          At the end of a match, each robot is awarded <strong>two</strong> additional points for each tin can it is carrying.
+        <li>
+          A robot is deemed to have passed a scoring line when the back of the robot passes the line.
+        <li>
+          Cases of a robot passing backwards over a line are offset against future crossings forward of a line. That is, if a robot crosses over two track boundaries backwards, it will need to cross two track boundaries forwards before it can gain any more scoring line points.
```

Comment:
This is a little bit wordy, would be good if we can optimise this down a bit?

Reply:
Perhaps we just remove the example?

```suggestion
          Cases of a robot passing backwards over a line are offset against future crossings forward of a line.
```